apps/web/src/pages/Pools/ViewPoolPage/ViewPoolPage.tsx

```diff
@@ -1,818 +1,393 @@
-    return skill.category === SkillCategory.Behavioural;
-  });
+  const { roleAssignments } = useAuthorization();
+  const poolName = getFullPoolTitleHtml(intl, pool);
+  const advertisementStatus = getAdvertisementStatus(pool);
+  const advertisementBadge = getPoolCompletenessBadge(advertisementStatus);
+  const assessmentStatus = "incomplete" as PoolCompleteness;
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const assessmentBadge = getPoolCompletenessBadge(assessmentStatus);
+  const processBadge = getProcessStatusBadge(pool.status);
+  const isAdmin = checkRole([ROLE_NAME.PlatformAdmin], roleAssignments);
 
-  const nonEssentialOccupationalSkills = pool.nonessentialSkills?.filter(
-    (skill) => {
-      return skill.category === SkillCategory.Technical;
-    },
-  );
-
-  const nonEssentialTransferableSkills = pool.nonessentialSkills?.filter(
-    (skill) => {
-      return skill.category === SkillCategory.Behavioural;
-    },
-  );
-
-  const languageRequirement = pool.language
-    ? intl.formatMessage(getLanguageRequirement(pool.language))
-    : notProvided;
-
-  const securityClearance = pool.securityClearance
-    ? intl.formatMessage(getSecurityClearance(pool.securityClearance))
-    : notProvided;
-
-  const screeningQuestions = pool?.screeningQuestions?.filter(notEmpty) || [];
-
-  const relativeToAbsoluteURL = (path: string): string => {
-    const { host, protocol } = window.location;
-    return `${protocol}//${host}${path}`;
-  };
-
-  let closingStringLocal;
-  let closingStringPacific;
+  let closingDate = "";
   if (pool.closingDate) {
     const closingDateObject = parseDateTimeUtc(pool.closingDate);
-    closingStringLocal = relativeClosingDate({
-      closingDate: closingDateObject,
-      intl,
-    });
-    closingStringPacific = relativeClosingDate({
-      closingDate: closingDateObject,
+    closingDate = formatDate({
+      date: closingDateObject,
+      formatString: DATE_FORMAT_STRING,
       intl,
-      timeZone: "Canada/Pacific",
     });
-  } else {
-    closingStringLocal = "";
-    closingStringPacific = "";
   }
 
+  const commonDialogProps = {
+    poolName,
+    isFetching,
+  };
+
   const pageTitle = intl.formatMessage({
-    defaultMessage: "View pool",
-    id: "vINfxJ",
+    defaultMessage: "Process information",
+    id: "IWWSkL",
     description: "Page title for the individual pool page",
   });
 
+  const pageSubtitle = intl.formatMessage({
+    defaultMessage: "Manage and view information about your process. ",
+    id: "pM43Xu",
+    description: "Subtitle for the individual pool page",
+  });
+
+  /** TO DO: Replace this message and link with
+   * appropriate values once assessment plan feature
+   * has been developed (#7916)
+   */
+  const comingSoon = intl.formatMessage({
+    defaultMessage: "Coming soon",
+    id: "d45Ihy",
+    description:
+      "MEssage displayed when a feature is in development and not ready yet",
```

Comment:
Don't forget to re-generate the id!
```suggestion
      "Message displayed when a feature is in development and not ready yet",
```